versions.tfvars.json

```
@@ -1,7 +1,7 @@
 {
   "armonik_versions": {
     "armonik":       "2.21.0",
-    "infra":         "0.8.0",
+    "infra":         "0.9.0",
```

https://github.com/aneoconsulting/ArmoniK.Infra/releases/tag/0.10.0-pre1-00b16b1
```suggestion
    "infra":         "0.10.0-pre1-00b16b1",
```